dotnet-conf-2022/index.html

```diff
@@ -0,0 +1,592 @@
+			</div>
+			</div>
+		</section>
+		<!-- End Organizers -->
+		<!-- Start Register  -->
+		<section id="mu-register">
+			<div class="container">
+				<div class="row">
+					<div class="col-md-12">
+						<div class="mu-register-area">
+							<div class="mu-title-area">
+								<h2 class="mu-title">Register </h2>
+							</div>
+
+							<div class="mu-register-content">
+								<tito-widget event="bdotnetug/dotnet-conf-2022"></tito-widget>
+							</div>
+
+						</div>
+					</div>
+				</div>
+			</div>
+		</section>
+		<!-- End Register -->
+		<!-- Start Code of Conduct -->
+		<section id="mu-about">
+			<div class="container">
+				<div class="row">
+					<div class="col-md-12">
+						<div class="mu-about-area" style="padding-top: 0px;">
+							<div class="mu-title-area">
+								<h2 class="mu-title">Code of Conduct</h2>
+							</div>
+							<!-- Start Feature Content -->
+							<div class="row">
+								<div class="col-md-12">
+									<h3>Attribution</h3>
+									<p>
+										Our User Groups (BDotNet and Dot Net Learners House) are dedicated to providing
+										a respectful,
+										harassment-free community for everyone. We do not tolerate harassment or
+										bullying of any community member in any form.
+										To clarify expected behaviour in our communities we have adopted the <a
+											style="color: #323eee;" href="https://www.contributor-covenant.org"
+											target="_blank">Contributor Covenant</a>, Code of Conduct version 2.0,
+										available at <a style="color: #323eee;"
+											href="https://www.contributor-covenant.org/version/2/0/code_of_conduct.html"
+											target="_blank">https://www.contributor-covenant.org/version/2/0/code_of_conduct.html</a>.
+										While attending the event, you must follow this Code of Conduct. If you have any
+										questions or feedback, please contact one of the organizers.
+									</p>
+								</div>
+							</div>
+							<!-- End Feature Content -->
+						</div>
+					</div>
+				</div>
+			</div>
+		</section>
+		<!-- End Start Code of Conduct -->
+		<!-- Start Sponsors -->
+		<section id="mu-sponsors">
+			<div class="container">
+				<div class="row">
+					<div class="col-md-12">
+						<div class="mu-sponsors-area">
+							<div class="mu-title-area">
+								<h2 class="mu-title">Sponsored by</h2>
+								<p>Thanks a ton to our sponsors for the awesome goodies and support!!!</p>
+							</div>
+
+							<!-- Start spnonsors brand logo -->
+
+							<div class="mu-sponsors-content">
+								<div class="row">
+									<div class="col-md-3">
+										<a href="https://www.microsoft.com" target="_blank">
+											<img src="assets/images/Microsoft.png" />
+										</a>
+									</div>
+									<div class="col-md-3">
+										<a href="https://dotnetfoundation.org/" target="_blank">
+											<img src="assets/images/dotnetfoundationhorizontal.svg" />
+										</a>
+									</div>
+									<div class="col-md-3">
```

Comment:
Tech platform can be removed from sponsors

Reply:
Tech platform removed from sponsors